### PR TITLE
Add fallback options to WebSockets guide

### PR DESCRIPTION
**Pull request by marklewin**
## Description

This addresses the following JIRA ticket: https://nexmoinc.atlassian.net/browse/DEVX-2554

Note that I have substantially rewritten the text provided by Victor. Please ensure that the meaning is unchanged.

### CODE REVIEW

File: _documentation/en/voice/voice-api/guides/websockets.md [hunk 1]
```diff
@@ -157,6 +157,126 @@ You can send audio back into the call by writing binary messages to the WebSocke
...
+  "uuid": "aaaaaaaa-bbbb-cccc-dddd-0123456789ab",
+  "conversation_uuid": "CON-aaaaaaaa-bbbb-cccc-dddd-0123456789ab",
+  "status": "unanswered",
+  "timestamp": "2020-03-31T12:00:00.000Z"
+}
+```
+
+### Websocket disconnected
+
+If the WebSocket connection is terminated by the application, your webhook will receive an event with `status` of `disconnected`:
```
Comment (vshisterov) [inline _documentation/en/voice/voice-api/guides/websockets.md:209]: Please add the information that one can respond with NCCO to this `disconnected` event callback to make some fallback action (reconnection, transfer, whatever suitable for the use-case).

Comment (marklewin) [reply _documentation/en/voice/voice-api/guides/websockets.md:209]: Victor - I have described the ability to respond to an NCCO for either can't connect/disconnect at the top of this section?

File: _documentation/en/voice/voice-api/guides/websockets.md [hunk 2]
```diff
@@ -157,6 +157,126 @@ You can send audio back into the call by writing binary messages to the WebSocke
...
+        "type": "websocket",
+        "uri": "wss://example.com/socket",
+        "content-type": "audio/l16;rate=16000"
+      }
+    ]
+  }
+]
+```
+
+Our API will `POST` the event to your webhook at `eventURL`. You can optionally return a new NCCO in the response with the required fallback actions. For example, if you cannot establish a connection, you might want to play a message to the caller or transfer the call.
```
Comment (benhayehudi) [inline _documentation/en/voice/voice-api/guides/websockets.md:187]: Is the default `POST` and is it user definable in their application settings?

File: _documentation/en/voice/voice-api/guides/websockets.md (hunk 1 above)
Comment (vshisterov) [reply _documentation/en/voice/voice-api/guides/websockets.md:209]: Ok, I'm just missing some logical pause/switch between `disconnected` payload (below) and the next phrase "if the disconnection is intentional...". One may understand it as "you should do a `PUT` request after receiving the `disconnected`". But probably it's just me :)

Comment (marklewin) [reply _documentation/en/voice/voice-api/guides/websockets.md:209]: I'm sorry @vshisterov , but I'm not sure I understand. Maybe I've got the wrong end of the stick entirely with this and I'm sure I could do a better job of explaining it with your help!

This is how I understand it:

- If the connection is dropped by the app, you get a `disconnected` event and you can fallback by providing a new NCCO.
- However, this happens whether the disconnection was unintentional (due to some problem) or you intentionally disconnected the websocket (as part of your business logic)
- You want to be able to fallback _only when the disconnect is unintentional_ , so a better approach to forcing a disconnect is to do a `PUT` request to hang up. If you do that, you can rely on the `disconnect` event being an unintentional disconnection and responding appropriately.

Is my understanding correct? And, if so, would something like the above better explain it?

File: _documentation/en/voice/voice-api/guides/websockets.md (hunk 2 above)
Comment (marklewin) [reply _documentation/en/voice/voice-api/guides/websockets.md:187]: Is this something you can advise on please, @vshisterov ?

File: _documentation/en/voice/voice-api/guides/websockets.md (hunk 1 above)
Comment (vshisterov) [reply _documentation/en/voice/voice-api/guides/websockets.md:209]: @marklewin yes it's absolutely correct 👍 
And I guess it'd be very helpful to add this explanation, it sounds super clear to me.

File: _documentation/en/voice/voice-api/guides/websockets.md (hunk 2 above)
Comment (vshisterov) [reply _documentation/en/voice/voice-api/guides/websockets.md:187]: Yes, default `eventMethod` is POST and can be overridden in the connect action, see 
https://developer.nexmo.com/voice/voice-api/ncco-reference#connect -> `eventMethod`.

Comment (benhayehudi) [reply _documentation/en/voice/voice-api/guides/websockets.md:187]: That's what I thought :) In that case, I think we should add a caveat that Our API will by default `POST` the event, unless the HTTP verb was overridden in the `connect` action.

File: _documentation/en/voice/voice-api/guides/websockets.md (hunk 1 above)
Comment (marklewin) [reply _documentation/en/voice/voice-api/guides/websockets.md:209]: Many thanks!

File: _documentation/en/voice/voice-api/guides/websockets.md (hunk 2 above)
Comment (marklewin) [reply _documentation/en/voice/voice-api/guides/websockets.md]: 👍

**Comment by benhayehudi** [comment]
I think this is a really good shape. @marklewin, thank you! @vshisterov, can you give it another look through before we merge?

**Pull request merged**